spec/controllers/users_controller_spec.rb

```diff
@@ -55,6 +75,35 @@
         expect(UserMailer.deliveries.last.body).to have_content("Welcome to myFlix")
       end
     end
+
+    context "when user was invited" do
+      let(:bob) { Fabricate(:user) }
+      let(:alice) { Fabricate.attributes_for(:user, full_name: "Alice Black", invite_token: "abcd") }
```

Comment:
This is attributes hash not User object. I would rather be more explicit like `alice_attributes`.  Because `bob` is a User object and `alice` is a hash.